test/extended/util/collect.go

```diff
@@ -70,6 +73,8 @@ func RunDiagnose(dir string) error {
 		vmCommand("KUBECONFIG=/opt/kubeconfig oc describe nodes", "describe-nodes.txt"),
 		vmCommand("KUBECONFIG=/opt/kubeconfig oc get pods --all-namespaces", "get-pods.txt"),
 		vmCommand("KUBECONFIG=/opt/kubeconfig oc describe pods --all-namespaces", "describe-pods.txt"),
+		vmCommand("KUBECONFIG=/opt/kubeconfig oc get mcp", "get-mcp.txt"),
```

Comment:
@guillaumerose can we also add `oc get mc`  no need to describe otherwise it will leak pull secret same with `mcp` better not to describe it?

Reply:
yes done!